packages/types/src/jwt.ts

```diff
@@ -1,3 +1,5 @@
+import { MembershipRole } from '.';
```

Comment:
🙃 This import doesn't read nicely. Can you please change it to the full path instead?

```suggestion
import { MembershipRole } from './organizationMembership';
```